lms/djangoapps/support/static/support/jsx/errors_list.jsx

```diff
@@ -14,8 +14,8 @@ class ShowErrors extends React.Component {
         <div className="alert alert-danger" role="alert">
           <strong>{gettext('Please fix the following errors:')}</strong>
           <ul>
-            {this.props.errorList.map(error =>
-              <li>{error}</li>,
+            {this.props.errorList.map((error, i) =>
+              <li key={i}>{error}</li>,
```

Comment:
Is it a good idea to use the index as a key?

Reply:
the key just have to be unique for each element in the loop, so for this instance the index will work.

Reply:
It will work but using the index as a key impacts the performance negatively in case the list is dynamic i.e will be added or removed from the list.